packages/blade/src/components/Card/Card.tsx

```diff
@@ -83,12 +83,17 @@ export type CardProps = {
    * Sets the height of the card
    */
   height?: BoxProps['height'];
+    /**
+    Stops UI breakage when height is 100% 
+   */
+  minHeight?: BoxProps['minHeight'];
```

Comment:
1. Formatting
2. Can update text to maybe just "sets minimum height of the card"
3. Can add min width also

Reply:
Thanks !
I'll make the changes asap .